### PR TITLE
Smari navbar fix

### PR DESCRIPTION
**Pull request by smaristeinar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,12 +20,16 @@
                 <span></span>
                 <span></span>
             </div>
+            <div id="line">
```
Comment (postmodernistx) [inline index.html]: I assume "id" will be used for some JavaScript later on? Otherwise change to the more reusable "class".

File: styling.css [hunk 2]
```diff
@@ -1 +1,446 @@
...
+* {
```
Comment (postmodernistx) [inline styling.css:1]: Are you using different output formats in the group? Consider unifying them to the same standard. Is this a generated file?

**Review by postmodernistx — APPROVED** [review]
Looks good!

File: index.html (hunk 1 above)
Comment (smaristeinar) [reply index.html]: on it

File: styling.css (hunk 2 above)
Comment (smaristeinar) [reply styling.css:1]: different outputs, planed on fixing it later.

**Pull request merged**